torch_xla/csrc/ops/ops.cpp

```diff
@@ -295,6 +295,32 @@ NodePtr AdaptiveAvgPool2dBackward(const Value& grad_output,
                             std::move(lower_fn));
 }
 
+NodePtr ComparisonOp(c10::Symbol kind, const Value& input, const Value& other) {
+  auto lower_fn = [kind](const ir::Node& node,
+                         ir::LoweringContext* loctx) -> ir::XlaOpVector {
+    xla::XlaOp xla_input = loctx->GetOutputOp(node.operand(0));
+    xla::XlaOp xla_other = loctx->GetOutputOp(node.operand(1));
+    xla::XlaOp xla_output = BuildComparisonOp(kind, xla_input, xla_other);
+    return node.ReturnOp(xla_output, loctx);
+  };
+  auto lower_for_shape_fn =
```

Comment:
We do not need this.
The shape for a compare is the same as input.

Reply:
With the element type changed to `PRED` though, which is why I did it this way. But I can do it the other way around.

Reply:
Done.